src/pivot-table/hooks/use-selections-model.ts

```diff
@@ -140,7 +136,10 @@ export default function useSelectionsModel(
     [selections, isLocked, selectedPivotCells, selectedField],
   );
 
-  const isSelected = useCallback((cell: Cell) => selectedPivotCells.has(cell), [selectedPivotCells]);
+  const isSelected = useCallback(
+    (cell: Cell) => !isLocked(cell) && selectedPivotCells.has(cell.ref.qElemNo),
```

Comment:
why not check isActive here, so we don't have do to that in dimensionValue?

Reply:
Genius! Will fix